src/core/qgis.h

```diff
@@ -1248,6 +1248,26 @@ class CORE_EXPORT Qgis
     };
     Q_ENUM( AngularDirection )
 
+    /**
+     * Transaction mode.
+     *
+     * \since QGIS 3.24
+     */
+    enum class TransactionMode : int
+    {
+      None = 0, /*!< Edits are buffered locally and sent to the provider when toggling
```

Comment:
Sipify doesn't support multiline documentation like this -- you have to make it all one giant single line.

Reply:
can it be added before the enum value as block comment=

Reply:
Not without sipify modifications 😱😜